pubsub/rocketmq/rocketmq.go

```diff
@@ -249,8 +249,14 @@ func (r *rocketMQ) adaptCallback(topic, consumerGroup, mqType, mqExpr string, ha
 	return func(ctx context.Context, msgs ...*primitive.MessageExt) (mqc.ConsumeResult, error) {
 		success := true
 		for _, v := range msgs {
+			attributes := map[string]interface{}{
```

Comment:
is this the only place where subject is added?

Reply:
Time and dataschema can also be added, but these are optional, including subject